File: example/src/androidTest/java/com/github/bassaer/example/ImageViewDrawableMatcher.java
package com.github.bassaer.example;

import android.view.View;

import org.hamcrest.Matcher;

/**
 * A matcher for Espresso that checks if an ImageView has a drawable
 * Created by nakayama on 2017/07/02.
 */

public class ImageViewDrawableMatcher {
    public static Matcher<View> withDrawable(final int resourceId) {
        return new DrawableMatcher(resourceId);
    }
}

File: example/src/androidTest/kotlin/com/github/bassaer/example/matcher/ImageViewDrawableMatcher.kt
package com.github.bassaer.example.matcher

import android.view.View
import org.hamcrest.Matcher

/**
 * A matcher for Espresso that checks if an ImageView has a specified drawable
 * Created by nakayama on 2017/07/02.
 */

object ImageViewDrawableMatcher {
    @JvmStatic
    fun withDrawable(resourceId: Int): Matcher<View> {
        return DrawableMatcher(resourceId)
    }
}
